deb/common/control

```diff
@@ -35,9 +35,9 @@ Depends: containerd.io (>= 1.4.1),
 Recommends: apparmor,
             ca-certificates,
             docker-ce-rootless-extras,
-            git,
```

Comment:
Following the discussion, looks like this one should stay, correct?

Reply:
Yeah, I guess. Invoking `git clone` daemon-side (potentially with root permissions, if we aren't super careful) still feels really, really wrong to me, but I'll update.